ccmbioinfo/lfs
# CARDS-1592: New Proms questionnaire SF12; CARDS-1594: questionnaire set for Mood Disorders

Draft until #825 is merged, as the new questionnaire SF12 relies on the `QuestionMatrix` component.

**Includes:**
* [CARDS-1592](https://phenotips.atlassian.net/browse/CARDS-1592) - _Proms: New questionnaire: SF-12_
* [CARDS-1594](https://phenotips.atlassian.net/browse/CARDS-1594) - _Proms: New Questionnaire set definition for Mood Disorders_
  * A basic Mood questionnaire set had already been developed (#910)
  * The SF-12 questionnaire was added to the Mood questionnaire set
  * Column definitions were added to all steps for proper configuration of the Mood dashboard

**Testing**:
* As Admin, create a future visit with the `Mood` questionnaire set
  * Check that 4 surveys are automatically created: SF12 (the new questionnaire), PHQ9, GAD7, AUDITC
* As Patient, fill out the surveys for that visit
  * Check that SF12 looks and behaves well for entering data on both wide and narrow screens
  * On the Review screen, check that all entered data is displayed for SF12 and that it is easy to read
* As Admin, create a new clinic configuration, and set Mood as the questionnaire set for that clinic
  * Navigate to the clinician dashboard and check that the survey boxes display the data as expected (prior to these changes, the survey boxes for Mood surveys would always be empty)
  * Navigate to the filled out SF12 survey and check that it looks good on both wide and narrow screens in view mode
    * Go to Print Preview, generate the PDF. Check that the PDF contains all the entered data, and that it is well formatted and easy to read.
